neo/Network/P2P/Payloads/GetFullBlocksPayload.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class GetFullBlocksPayload : ISerializable
+    {
+        public const ushort MaxBlocksCount = 500;
+        public uint IndexStart;
+        public short Count;
```

Comment:
`ushort`

Reply:
I tried to stay close to 
https://github.com/neo-project/neo/blob/1c84f7cf9f211f386107677ce28b26de26042677/neo/Network/P2P/Payloads/GetBlocksPayload.cs#L10

Reply:
They are different. `GetBlocks` is to get the hashes of blocks. But `GetBlockData` is to get the blocks. Blocks are much larger then hashes.